frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/ticketImageContent/ticketImage/ticketImage.component.tsx

```diff
@@ -34,7 +35,7 @@ export const TicketImage = ({ value, onChange, onBlur, disabled: inputDisabled,
 	useEffect(() => { setTimeout(() => { onBlur?.(); }, 200); }, [value]);
 
 	return (
-		<InputContainer disabled={disabled} {...props}>
+		<ViewerInputContainer disabled={disabled} {...props}>
```

Comment:
![image](https://github.com/3drepo/3drepo.io/assets/62481250/a7aac4be-792e-4723-b01f-dfeadc38266c)
Should the user be able to set an image here, but not a screenshot? Or might that be too complex